src/utils/index.ts

```diff
@@ -49,6 +50,32 @@ export interface Route {
   interceptor?: (req: Request, res: Response, next: NextFunction) => void;
 }
 
+async function pgTx<T>(pool: Pool, cb: (client: ClientBase) => Promise<T>): Promise<T> {
```

Comment:
maybe `pgSnapshot` since there are no updates, (in case of always rolling back)

Reply:
good point